bitnami/clickhouse/24.3/debian-12/rootfs/opt/bitnami/scripts/libclickhouse.sh

```diff
@@ -243,7 +243,7 @@ clickhouse_start_bg() {
         error "ClickHouse failed to start"
         exit 1
     fi
-    wait_for_log_entry "Application: Ready for connections" "$log_file"
+    wait_for_log_entry "Ready for connections" "$log_file"
```

Comment:
I think this change is not necessary. I mounted a sample script in the container in all 3 branches (24, 24.3 and 24.8)

```
#!/bin/bash

echo "Jota test"
```

```
# Copyright Broadcom, Inc. All Rights Reserved.
# SPDX-License-Identifier: APACHE-2.0

services:
  clickhouse:
    image: docker.io/bitnami/clickhouse:24
    environment:
      - ALLOW_EMPTY_PASSWORD=yes
    ports:
      - '8123:8123'
    volumes:
      - clickhouse_data:/bitnami/clickhouse
      - ./test.sh:/docker-entrypoint-initdb.d/test.sh
volumes:
  clickhouse_data:
    driver: local

```

and they all included the output of my script and the line in the log file 

```
...
clickhouse-1  | Jota test
...
clickhouse-1  | 2024.12.17 14:44:03.719098 [ 723 ] {} <Information> Application: Ready for connections.
...
```

Could you please take a look at it?

Reply:
**To reproduce this issue, you need to enable logging in JSON format**, which is something most people will want to enable if they want index the logs in a monitoring tool like kibana.
How to enable JSON logging it and more details about the error are explained in the description of linked issue. To summarize it, you should mount this config file in `/bitnami/clickhouse/etc/conf.d/default/config.xml`:
```
<clickhouse replace="true">
    <logger>
        <level>information</level>
        <formatting>
            <type>json</type>
        </formatting>
    </logger>
    <display_name>ch-1S_1K</display_name>
    <listen_host>0.0.0.0</listen_host>
    <http_port>8123</http_port>
    <tcp_port>9000</tcp_port>
</clickhouse>
```

Reply:
Thanks for the info, I could reproduce the issue.